docs/Using Fleet/segment-hosts.md

```diff
@@ -66,7 +66,7 @@ To automatically add hosts to a team in Fleet, check out the ["Adding hosts" doc
 
 ## Transfer hosts to a team
 
-Hosts can be transferred to a different team they've has been enrolled to Fleet.
+Any Host enrolled in Fleet and assigned to a team can be transferred to a different team.
```

Comment:
How about we just cut this line instead? (Since it's sort of a repeat of what's said in the section above.)
```suggestion
```
Reasoning: Hosts can be enrolled without a team, and this potentially implies only hosts that are already on teams can be transferred.

Reply:
Agree.

Reply:
I'm not sure what the protocol is. Do you want me to delete the line & like submit again? Sorry for being dumb. :)